roles/monitoring-common/defaults/main.yml

```diff
@@ -11,4 +11,4 @@ monitoring:
     vhost: /sensu
     username: sensu
     password: sensu
-  cluster_prefix: "ip a | grep :ucarp > /dev/null &&"
\ No newline at end of file
+  cluster_prefix: "ip a | grep :ucarp > /dev/null"
```

Comment:
Something else is broken in the neutron-free-fips check then, because all that gets written out is the cluster_prefix. An `&&` with no following command is syntactically invalid. What did you mean to do?